pom.xml

```diff
@@ -62,12 +62,37 @@
 		<dependency>
 			<groupId>life.qbic</groupId>
 			<artifactId>data-model-lib</artifactId>
-			<version>2.0.0</version>
+			<version>2.4.0</version>
 		</dependency>
 		<dependency>
 			<groupId>life.qbic.openbis</groupId>
 			<artifactId>openbis_api</artifactId>
 			<version>3-S253.0</version>
 		</dependency>
     </dependencies>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-enforcer-plugin</artifactId>
+				<version>3.0.0-M2</version>
+				<executions>
+					<execution>
+						<id>default-cli</id>
+						<phase>verify</phase>
+						<goals>
+							<goal>enforce</goal>
+						</goals>
+						<configuration>
+							<rules>
+								<dependencyConvergence/>
+								<banDuplicatePomDependencyVersions/>
+							</rules>
+							<fail>false</fail>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+		</plugins>
+	</build>
```

Comment:
What does this do? Should the change be described in the change log?

Reply:
good point

Reply:
ah, guess it's this: "Uses maven enforcer plugin to show depencency conflicts"
if you add this information to squash and merge I'm happy!

Reply:
@sven1103 I do not know what this actually does neither why it was added. Can you remember?